state-chain/runtime/src/lib.rs

```diff
@@ -1408,6 +1408,25 @@ impl_runtime_apis! {
 			all_prewitnessed_swaps
 		}
 
+		fn cf_scheduled_swaps(from: Asset, to: Asset) -> Vec<(Swap, BlockNumber)> {
```

Comment:
These should be consistent with the other interfaces, and therefore called base_asset and quote_asset.

Reply:
And the ordering should be enforced to match the pool.

Reply:
Right, I see a problem. I believe what we want (and what was discussed in discord before) from this sub is per pool scheduled swaps, otherwise the sub isn't very useful. Where as my (somewhat brief) reading of this, is this implementation is a sub of end to end scheduled swaps. The problem with that is LPs would have to do not insignificant amount of work to work out what the per pool scheduled swaps are.